features/__snapshots__/validate_image.snap

```diff
@@ -1944,6 +1955,7 @@ Results:
 [31m✕[0m [31m[Violation] main.reject_with_term[0m
   ImageRef: ${REGISTRY}/acceptance/image@sha256:${REGISTRY_acceptance/image:latest_DIGEST}
   Reason: Fails always (term2)
+  Term: [term2 term3]
```

Comment:
Idea: This output is for human consumption, so we could make it a little nicer by formatting a list something like this:

```suggestion
  Terms: term2, term3
```

(We could worry about ambiguity for terms with spaces or commas in them, but that feels like it will not be a problem in reality, i.e. an edge case we could ignore.)

Reply:
Thinking about how to do that: It might be a bit fiddly with the templating. I was thinking a helper in `internal/utils/templates.go`, either to output nice term list or to detect if term is a list of strings or a single string could be useful.